### PR TITLE
DEVOPS-58: Added new utility class to ping an endpoint every 2 second…

### PR DESCRIPTION
**Pull request by asharms26**
**NOTE: This PR creates a new "devops-util" folder for future use. If we have any utility classes or anything that may be of use, we can put it in this folder.**

Was originally used to test ZDD, but it can be used in the future.

CMD Arguments:

1. URL
2. Request Method (only GET, POST, PUT)
3. OAuth Token

### CODE REVIEW

**Comment by lenardw1** [comment]
Change the commit into from master to dev

**Comment by asharms26** [comment]
@lenardw1 Merge destination changed to "dev."

File: devops-util/DevopsRestTester/.idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,291 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ChangeListManager">
```
Comment (RussianPrussian) [inline devops-util/DevopsRestTester/.idea/workspace.xml]: How much of this xml stuf can be added into the .gitignore?

**Comment by RussianPrussian** [comment]
Yeah. this is all intellij stuff. Don't need it

**Pull request closed without merging**